.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+# Trigger this workflow on push or pull request
+on: [push, pull_request]
```

Comment:
Will one need to be triggered when a pull request is created? Ya need to push your changes to a branch anyway before one can be made anyway.

Reply:
_(Ignore review made. It was a misclick.)_

Reply:
Yes, one gets triggered upon push anyway, but that run doesn't show up on the pull request page. The pull request run is to make it easy for reviewers to see if the PR succeeds or fails to compile.

Reply:
Ah, ok. Fair enough.